requirements.txt

```diff
@@ -0,0 +1,10 @@
+flake8
```

Comment:
The reason the requirements.txt is empty in this repo is because the dependencies are managed in the setup.py.

If there are dependencies needed for development, they should go in here, but `python setup.py develop` will not reflect what you did.

Check out the setup.py for the current practice. Adding toil as an `extras_require` probably makes more sense.

https://github.com/DailyDreaming/workflow-service/blob/73a31f37a61a4a78a974512b63c961f7f584dd35/setup.py#L25

Reply:
Alright, will change it back!